Levi9.POS/Request/ProductSearchRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Levi9.POS.Domain.DTOs;
+using System.ComponentModel.DataAnnotations;
+
+namespace Levi9.POS.WebApi.Request
+{
+    public class ProductSearchRequest
+    {
+        public int Page { get; set; } = 1;
+        public string Name { get; set; }
+        [RegularExpression("^(name|id|globalId|availableQuantity)$", ErrorMessage = "The document type must be name,id,globalId,availableQuantity.")]
```

Comment:
"The document type must be name,id,globalId,availableQuantity." Is this correct? 
Since the property is OrderBy. 
Should be something like: "OrderBy value can be: name,id,globalId,availableQuantity"

Reply:
I changet to "The value of OrderBy can be: name, id, globalId, availableQuantity".